From 9c8111a1d09f216ee529eed4a2a2e559327c6a8a Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Fri, 14 Oct 2016 21:46:42 -0700
Subject: [PATCH 21/25] Settings: Add a RemotePreference for device-specific
 doze settings

 * If a device has its own panel, we should replace the item under
   display settings instead of having to clutter up the dashboard.
 * This uses the new RemotePreference API to allow the external
   app to update the view.
 * The action for this is "org.lineageos.settings.device.DOZE_SETTINGS".

Change-Id: I22fd7e6307b370807415ce2132bed4808b2befe1
---
 res/xml/display_settings.xml | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/res/xml/display_settings.xml b/res/xml/display_settings.xml
index bd22140a10..7dfe116d7c 100644
--- a/res/xml/display_settings.xml
+++ b/res/xml/display_settings.xml
@@ -117,6 +117,13 @@
         android:fragment="com.android.settings.display.AmbientDisplaySettings"
         settings:userRestriction="no_ambient_display" />
 
+    <lineageos.preference.RemotePreference
+        android:key="doze_device_settings"
+        android:title="@string/ambient_display_screen_title"
+        lineage:replacesKey="ambient_display">
+        <intent android:action="org.lineageos.settings.device.DOZE_SETTINGS" />
+    </lineageos.preference.RemotePreference>
+
     <!-- Hide night mode for now
     <ListPreference
         android:key="night_mode"
-- 
2.17.1

